Resources/Public/JavaScript/Test.js

```diff
@@ -0,0 +1,20 @@
+const menuConfig = {
+  title: "Order",
+  buttonText: "Send",
+  cancellable: true,
+};
+
+function createMenu(config) {
```

Comment:
Can we have some JS here that does something real for the Tea extension?

Reply:
ok :)